content/legal/acronym_guidelines/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Oracle/JCP Acronym Guidelines"
+date: 2019-06-26T16:10:38-04:00
+description: "Guidelines for the use of existing Oracle/JCP Acronyms.  Supplement to the Eclipse Foundation Guidelines for Eclipse Logos & Trademarks Policy for Oracle/JCP Acronyms"
```

Comment:
Where is this other document that this is a supplement to?

Reply:
Updated it to use the same link as this page: https://jakarta.ee/legal/trademark_guidelines/
That goes here: https://www.eclipse.org/legal/logo_guidelines.php